### DISCUSSION TITLE
Tooltip is not working in Mobile, Tablet (Touch device) Ultima Template

### DISCUSSION THREAD

**Discussion by alpesh88ww** [primefaces/community#3123] [PrimeNG]
We have purchased Prime-Ng Ultima Template (The issue is with the tooltip on PrimeNG's website itself — it's not related to any specific template.) for our website but we found that Tooltip is not working in Mobile, Tablet (Touch device).

I am expecting your help.

**Comment by AshishFreak** [comment]
Hello can you share Ultima theme ?

**Reply by alpesh88ww** [reply]
> Hello can you share Ultima theme ?

No dear. I can't do that - my company paid for it, I hope you understand.

**Comment by alpesh88ww** [comment]
> We have purchased Prime-Ng Ultima Template for our website but we found that Tooltip is not working in Mobile, Tablet (Touch device).
> 
> I am expecting your help.

**Is no one having this problem?**
Why has no one responded to this yet?
Even the team at PrimeNG hasn’t seen or acknowledged it. I’m not sure why this is being overlooked — I would really appreciate a response.
